bellbot_gui/scripts/gui2.py

```diff
@@ -37,6 +36,7 @@ def florence():
                 rospy.logerr("virtual keyboard not found, please install it by running:\nsudo apt-get install florence")
     except:
         try:
+            pass
```

Comment:
There is another line below `pass` that is run... Anyway agree cause confusion so I can comment it out.